linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -133,7 +133,7 @@ public static void main(String[] arguments)
       } catch (ModelBuildingException ex) {
         // When there's a pom.xml file that is not (indirectly) referenced by the root pom.xml,
         // Maven may fail to build the model. Such pom.xml can be ignored.
-        System.out.println("Ignoring bad model: " + path);
+        System.out.println("Ignoring bad model: " + path + ": " + ex.getMessage());
```

Comment:
do we want to System.out.println here? A logger would be better, or perhaps this case shouldn't be ignored.

Reply:
Now it uses logger.

Reply:
About ignoring this failure, the output in the comment below shows that target directory may contain pom.xml files. Linkage Monitor should not be stopped by them.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1035#issuecomment-553846199